### PR TITLE
Update sample app to support android 12

### PR DESCRIPTION
**Pull request by waelba**
- Update hms push library version to support android:exported flag using android 12 as target

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -61,7 +63,8 @@
         <activity
             android:name=".MainActivity"
             android:label="@string/app_name"
-            android:windowSoftInputMode="adjustResize">
+            android:windowSoftInputMode="adjustResize"
+            android:exported="true">
```
Comment (waelba) [inline app/src/main/AndroidManifest.xml:67]: Launch activity must be set to true otherwise we can't open the app

Comment (CedricBm) [reply app/src/main/AndroidManifest.xml:67]: ah, so the main application must always have at least one activity with exported "true"?

Comment (waelba) [reply app/src/main/AndroidManifest.xml:67]: yes

File: app/build.gradle [hunk 2]
```diff
@@ -66,7 +66,7 @@ android {
         implementation project(':dimelosdk-debug')
         implementation project(':glide-debug')
         implementation project(':volley-debug')
-        implementation 'com.huawei.agconnect:agconnect-core:1.3.1.300'
+        implementation 'com.huawei.hms:base:6.2.0.300'
```
Comment (waelba) [inline app/build.gradle:69]: this line fix the hms mutable flag bug on android 12 https://stackoverflow.com/questions/69225285/android-12-crash-huawei-services

**Comment by CedricBm** [comment]
Good to me.

**Pull request merged**